source/isaaclab/docs/CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 ---------
 
+0.46.3 (2025-09-17)
```

Comment:
Could you please update the `extension.toml` as well?

Reply:
Done in https://github.com/isaac-sim/IsaacLab/pull/3318/commits/148114e40f8752e423ebac36d584718c9cfdca20